editors/code/src/inlay_hints.ts

```diff
@@ -14,6 +14,8 @@ export function activateInlayHints(ctx: Ctx) {
                 return this.dispose();
             }
             if (!this.updater) this.updater = new HintsUpdater(ctx);
+
+            this.updater.syncCacheAndRenderHints();
```

Comment:
```suggestion
            if (this.updater) {
                this.updater.syncCacheAndRenderHints();
            } else {
                this.updater = new HintsUpdater(ctx);
            }
```
The new updater will sync cache and render hints in it's constructor, so it's better to avoid doing it 2 times.